app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import app.errors
+import datetime
```

Comment:
Order your imports according to PEP-8. Besides that, leave blank lines between different groups of imports. This comment relates to `app/main.py` as well.